cpp/CMakeLists.txt

```diff
@@ -474,7 +474,7 @@ if(BUILD_TESTS)
         # link_directoires (we should switch to target_link_directories once 3.13 becomes the
```

Comment:
Not your PR, but could you fix the typo and spell directories correctly here :-)

Reply:
Thanks, fixed in next commit.